app/frontends/workflow-ui.ts

```diff
@@ -269,7 +269,7 @@ function workItemRenderingFunctions(job: Job, isAdmin: boolean, requestUser: str
     workflowItemCreatedAt(): string { return this.createdAt.getTime(); },
     workflowItemUpdatedAt(): string { return this.updatedAt.getTime(); },
     workflowItemLogsButton(): string {
-      const isComplete = COMPLETED_WORK_ITEM_STATUSES.indexOf(this.status) > -1;
+      const isComplete = [WorkItemStatus.FAILED, WorkItemStatus.SUCCESSFUL].indexOf(this.status) > -1;
```

Comment:
Why did you need to make this change? I assume the other status is 'canceled'?

Reply:
Yes I'm assuming if it's canceled, the majority of the time we won't have logs to look at, nor will we care to look at them.

Reply:
The tricky thing is that there is no efficient way to check whether the logs button should be shown other than by looking at the status of the work item. The alternative would be to show the logs button for all of the canceled work items, and just return an error response for those that are missing.